apps/dsb-client-gateway-api/src/app/modules/topic/dto/index.ts

```diff
@@ -290,6 +290,15 @@ export class GetTopicSearchDto {
   })
   owner: string;
 
+  @IsString()
+  @IsNotEmpty()
+  @ApiProperty({
+    description: 'owner of the topic',
+    type: String,
+    example: 'torta.apps.eggplant.vege.iam.ewc',
+  })
+  version: string;
```

Comment:
Why it did not exist previously? Did Vikas remove this to solve issues related to Python Lib? 
@hejkerooo Are you aware of any previous details about this?